src/component/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Button({ name }) {
```

Comment:
### [Optional]
For good naming convention, you could use `buttonName` as props name instead of just `name`